config/locales/fr.yml

```
@@ -683,6 +683,13 @@ fr:
       update: Modifier
       submit: Envoyer
 
+  concerns:
+    access_control:
+      badge: Badge
+      key: Clé
+      key_loken: Clé Loken
```

```suggestion
      locken_key: Clé Locken
```

Also rename other in the concern code